### PR TITLE
DevKit updates for 3.x branch

### PR DESCRIPTION
**Pull request by SonataCI**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -41,7 +42,7 @@ matrix:
       env: TARGET=docs
     - php: '7.3'
       env: TARGET=lint
-    - php: '7.2'
+    - php: '7.1'
```
Comment (OskarStark) [inline .travis.yml]: Hmm 🤔

File: .travis.yml [hunk 2]
```diff
@@ -5,11 +5,12 @@
 branches:
   only:
     - master
-    - 2.x
+    - 3.x
 
 language: php
 
 php:
+  - '7.1'
```
Comment (OskarStark) [inline .travis.yml]: Hmm 🤔

File: .travis.yml (hunk 1 above)
Comment (greg0ire) [reply .travis.yml]: Oh that's weird… what happened?

Comment (OskarStark) [reply .travis.yml]: `3.x` wasn't managed by dev kit before, right?

Comment (greg0ire) [reply .travis.yml]: Ah yeah that's it!

**Comment by OskarStark** [comment]
Does somebody has time to rebase this branch?

**Comment by greg0ire** [comment]
Done :)

**Pull request merged**